ios/FluentUI/Generated/FluentUIStyle.generated.swift

```diff
@@ -0,0 +1,2958 @@
+/// Autogenerated file
+
+// swiftlint:disable all
+import UIKit
+
+public class Application {
+	@objc dynamic public class func preferredContentSizeCategory() -> UIContentSizeCategory {
+		return .large
+	}
+}
+
+/// Your view should conform to 'AppearaceProxyComponent'.
+public protocol AppearaceProxyComponent: class {
+    associatedtype ApperanceProxyType
+    var appearanceProxy: ApperanceProxyType { get }
+    var themeAware: Bool { get set }
+    func didChangeAppearanceProxy()
+}
```

Comment:
type, appearance?

Reply:
I didn't get your point here. Can you elaborate?